Binner/Library/Binner.Common/IO/CsvBOMImporter.cs

```diff
@@ -62,7 +50,7 @@ public async Task<ImportResult> ImportAsync(Project project, Stream stream, IUse
                 // read csv header
                 Header? header = null;
                 var headerRow = rows.First();
-                if (headerRow.StartsWith("#"))
+                if (headerRow.StartsWith("\"#\""))
```

Comment:
This should likely be `if (headerRow.StartsWith("#") || headerRow.StartsWith("\"#\""))` as many CSV formats don't require quoted headers